src/modules/Aieracast/index.tsx

```diff
@@ -438,7 +438,7 @@ export function Aieracast(): ReactElement {
                 scrollRef.current.scrollTo({ left: width, behavior: 'smooth' });
             }
         }
-        if (typeof scrollWidth === 'number') {
+        if (scrollWidth !== undefined) {
```

Comment:
not sure why you made this change, it shouldn't be anything besides a number, so I don't see how its a better condition